### PR TITLE
Update to Bevy 0.11

### PR DESCRIPTION
**Pull request by PikminGuts92**
Draft until `bevy_flycam` is updated ~~https://github.com/sburris0/bevy_flycam/pull/34~~ https://github.com/sburris0/bevy_flycam/pull/35

Changes
- Migrate to bevy 0.11
- Update `README` with updated syntax

### CODE REVIEW

**Comment by rlidwka** [comment]
> Draft until bevy_flycam is updated

bevy_flycam was updated yesterday, so is this PR all good?

**Comment by PikminGuts92** [comment]
> bevy_flycam was updated yesterday, so is this PR all good?

Change was merged in, yes. But crates.io release is not available yet. Otherwise, it's completely functional aside from example.

File: src/render/shadow.rs [hunk 1]
```diff
@@ -539,13 +539,14 @@ pub fn register_shadow(app: &mut App) {
...
             // Register as exclusive system because ordering against `bevy_render::view::prepare_view_uniforms` isn't possible otherwise.
+            Render,
             prepare_grid_shadow_views.in_set(RenderSet::Prepare),
         )
-        .add_system(queue_grid_shadows.in_set(RenderSet::Queue))
-        .add_system(queue_grid_shadow_bind_groups.in_set(RenderSet::Queue))
-        .add_system(queue_grid_shadow_view_bind_group.in_set(RenderSet::Queue));
+        .add_systems(Render, queue_grid_shadows.in_set(RenderSet::Queue))
+        .add_systems(Render, queue_grid_shadow_bind_groups.in_set(RenderSet::Queue))
+        .add_systems(Render, queue_grid_shadow_view_bind_group.in_set(RenderSet::Queue));
```
Comment (Hibyehello) [inline src/render/shadow.rs:549]: Couldn't these also be combined into one call?

**Review by Hibyehello — COMMENTED** [review]
I just have a couple thoughts on cleaning up all the add_systems

**Comment by PikminGuts92** [comment]
> I just have a couple thoughts on cleaning up all the add_systems

@Hibyehello Great suggestions, thanks! Those changes are now pushed up. I also went ahead and updated the `README` to use newer syntax too

**Comment by IceSentry** [comment]
@PikminGuts92 thanks for making this PR. I'll merge it as is and will replace flycam for a simple camera controller copied from bevy. I don't want this PR to be blocked just because of a third party library only used in the example

**Pull request merged**